src/System.Private.Xml/src/System/Xml/Xsl/Runtime/XmlCollation.Unix.cs

```diff
@@ -421,6 +420,18 @@ internal CultureInfo Culture
             }
         }
 
+        //-----------------------------------------------
+        //
+        //-----------------------------------------------
+
+        /// <summary>
+        /// Create a sort key that can be compared quickly with other keys.
+        /// </summary>
+        internal XmlSortKey CreateSortKey(string s)
+        {
+            throw new NotImplementedException();
```

Comment:
also you may use for now the implementation of Windows and just remove the part which change the sort key

---
In reply to: [86050030](https://github.com/dotnet/corefx/pull/13222#discussion_r86050030) [](ancestors = 86050030)